derive/src/multihash.rs

```diff
@@ -89,6 +93,17 @@ impl Hash {
             Multihash::wrap(#code, hasher.finalize()).unwrap()
         })
     }
+
+    fn code_reader(&self) -> TokenStream {
+        let ident = &self.ident;
+        let hasher = &self.hasher;
+        let code = &self.code;
+        quote!(Self::#ident => {
+            let mut hasher = #hasher::default();
+            io::copy(reader, &mut hasher)?;
+            Ok(Multihash::wrap(#code, hasher.finalize()).unwrap())
```

Comment:
Can you please add a comment why it is OK to use `unwrap()` here (or using an `expect()` which makes it clear?